tools.yml

```diff
@@ -2313,3 +2334,181 @@ tools:
     mem: 32
   wig_to_bigWig:
     mem: 10
+toolshed.g2.bx.psu.edu/repos/aaronquinlan/bedtools/bedtools_intersectbed_bam/.*:
+  mem: 40
+toolshed.g2.bx.psu.edu/repos/bgruening/antismash/.*:
+  cores: 10
+  mem: 90
+  env:
+    _JAVA_OPTIONS: -XX:MaxPermSize=2G -Xmx{int(mem)}G -Xms1G
+toolshed.g2.bx.psu.edu/repos/bgruening/bismark/bismark_bowtie/.*:
+  cores: 6
+  mem: 30
+toolshed.g2.bx.psu.edu/repos/bgruening/bismark/bismark_bowtie2/.*:
+  cores: 6
+  mem: 30
+toolshed.g2.bx.psu.edu/repos/bgruening/deeptools/deeptools_bamCorrelate/.*:
+  cores: 10
+  mem: 90
+toolshed.g2.bx.psu.edu/repos/bgruening/openduck_run_smd/openduck_run_smd/.*:
+  gpus: 1
+  cores: 1
+  env:
+    CUDA_VISIBLE_DEVICES: 0
+    OPENDUCK_GPU_PARAM: --gpu-id 1
```

Comment:
Just raising this for discussion - how should nodes with multiple gpus work? My understanding is that this would always force the first device only. That seems like a reasonable default for a shared database, but just wondering whether there are alternative solutions.

I don't know what the `--gpu-id` param is, but should that also be 0 to match the exposed `CUDA_VISIBLE_DEVICES` device number?

Reply:
For `CUDA_VISIBLE_DEVICES`, thats a very good question. And I do not have a good answer for that. We only provide VMs with one GPU for that exact reason. We split the GPUs on the virtual level, so we do not need to care about it on the tool level.

Reply:
Makes sense. Can always be overridden downstream, so this seems like a good default.